administration/src/mui-modules/download/DownloadPage.tsx

```diff
@@ -0,0 +1,60 @@
+import { Box, Typography } from '@mui/material'
+import { styled } from '@mui/system'
+import React, { ReactElement, useContext } from 'react'
+import { useTranslation } from 'react-i18next'
+import { Link } from 'react-router'
+
+import AppStoreLinks from '../../bp-modules/components/AppStoreLinks'
+import { ProjectConfigContext } from '../../project-configs/ProjectConfigContext'
+import { getBuildConfig } from '../../util/getBuildConfig'
+
+const AppIcon = styled('img')({
+  height: 100,
+  width: 100,
+  border: '1px solid rgba(0, 0, 0, 0.2)',
+  borderRadius: '20%',
+  marginBottom: '1rem',
+})
+
+const linkStyle = {
+  color: 'inherit',
+  textDecoration: 'none',
+  fontSize: '1rem',
+}
+
+const StyledAnchorLink = styled('a')({
+  ...linkStyle,
+  textDecoration: 'underline',
+})
+
+const StyledLink = styled(Link)({
+  ...linkStyle,
+})
+
+const DownloadPage = (): ReactElement => {
+  const { t: regionSettingsTranslations } = useTranslation('regionSettings')
+  const { t: activationTranslations } = useTranslation('activation')
+
+  const config = useContext(ProjectConfigContext)
+  const { ios, android } = getBuildConfig(window.location.hostname)
+
+  return (
+    <Box display='flex' flexDirection='column' alignItems='center' mx='auto' mt={10}>
+      <AppIcon alt={`${config.name} app icon`} src={`/icons/${config.projectId}.png`} aria-hidden />
+      <Typography variant='h5' component='h1'>
+        {config.name}
+      </Typography>
+      <StyledAnchorLink href='https://tuerantuer.de/digitalfabrik'>Tür an Tür - Digitalfabrik gGmbH</StyledAnchorLink>
```

Comment:
❓ it was not actually mentioned in the issue description but it probably makes sense to open this link with target blank, since its external?

Please mind security issues with target blank links, as described here
https://mui.com/material-ui/react-link/#security